src/components/file-uploader/_file-uploader.scss

```diff
@@ -28,14 +29,13 @@
     @include typescale('zeta');
     @include line-height('body');
     color: $text-02;
-    margin-top: 0.25rem;
```

Comment:
I'm kind of scratching the surface, but - Is removal of this line intentional?

Reply:
Yep, it’s intentional, the spacing on this component was updated, I can link the issue with the spec from Anna later.

Reply:
That's fine, as long as it's intentional I'm okay with merging!